src/main.py

```
@@ -285,4 +285,5 @@ def tick():
             #distance_overlay,
             raycasting_overlay
         ],
+        sys.argv[1] if len(sys.argv) > 1 else None
```

Ruff (0.14.10)

`sys` may be undefined, or defined from star imports

(F405)